public/app/features/gops/configuration-tracker/hooks/irmHooks.ts

```diff
@@ -0,0 +1,380 @@
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-oncall-app/settings',
+                queryParams: { tab: 'ChatOps', chatOpsTab: 'Slack' },
+              },
+              label: 'Connect',
+              urlLinkOnDone: {
+                url: '/a/grafana-oncall-app/settings',
+                queryParams: { tab: 'ChatOps' },
+              },
+              labelOnDone: 'View',
+              done: chatOpsConnections.is_chatops_connected,
+            },
+          },
+          {
+            title: 'Connect your Messaging workspace to Incident',
+            description:
+              'Automatically create an incident channel and manage incidents directly within your chat environment.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-incident-app/integrations/grate.slack',
+              },
+              label: 'Connect',
+              urlLinkOnDone: {
+                url: '/a/grafana-incident-app/integrations',
+              },
+              done: incidentPluginConfig?.isChatOpsInstalled,
+            },
+          },
+          {
+            title: 'Add Messaging workspace channel to OnCall Integration',
+            description: 'Select ChatOps channels to route notifications',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-oncall-app/integrations/',
+              },
+              label: 'Add',
+              urlLinkOnDone: {
+                url: '/a/grafana-oncall-app/integrations/',
+              },
+              labelOnDone: 'View',
+              done: chatOpsConnections.is_integration_chatops_connected,
+            },
+          },
+        ],
+      },
+      {
+        title: 'Test your configuration',
+        description: '',
+        steps: [
+          {
+            title: 'Send OnCall demo alert',
+            description: 'In the integration page, click Send demo alert, to review your notification',
+            button: {
+              type: 'dropDown',
+              label: 'Select integration',
+              options: onCallOptions,
+              onClickOption: (value) => onIntegrationClick(value, '/a/grafana-oncall-app/integrations/'),
+            },
+          },
+          {
+            title: 'Create Incident drill',
+            description: 'Practice solving an Incident',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-incident-app',
+                queryParams: { declare: 'new', drill: '1' },
+              },
+              label: 'Start drill',
+            },
+          },
+        ],
+      },
+    ],
+  };
+  const { stepsDone, totalStepsToDo } = essentialContent.sections.reduce(
+    (acc, section) => {
+      const stepsDone = section.steps.filter((step) => step.button.done).length;
+      const totalStepsToForSection = section.steps.reduce(
```

Comment:
Isn't `totalStepsForSection` just `section.steps.lenght`?

Reply:
there are some steps that are not count for this number (done can be undefined, for some steps where we are not checking anything)